src/drivers/opengles/lv_opengles_window.c

```diff
@@ -0,0 +1,438 @@
+    if(disp == NULL) {
+        return;
+    }
+    lv_opengles_window_t * dsc = lv_display_get_driver_data(disp);
+    if(dsc == NULL) {
+        return;
+    }
+
+    glfwSetWindowShouldClose(dsc->window, GLFW_TRUE);
+    lv_display_send_event(disp, LV_EVENT_DELETE, NULL);
+}
+
+/**
+ * Handler to update texture
+ */
+static void window_update_handler(lv_timer_t * t)
+{
+    //lv_display_t * disp = lv_timer_get_user_data(t);
+    lv_display_t * disp = lv_display_get_default();
+    if(disp == NULL) {
+        return;
+    }
+    lv_opengles_window_t * dsc = lv_display_get_driver_data(disp);
+    if(dsc == NULL) {
+        return;
+    }
+
+    if(!glfwWindowShouldClose(dsc->window)) {
+        lv_opengles_render_clear();
+        lv_opengles_texture_update(dsc->fb1, disp->hor_res, disp->ver_res);
+
+        lv_opengles_shader_bind();
+        lv_opengles_shader_set_uniform4f("u_Color", 0.0f, 0.3f, 0.8f, 1.0f);
+        lv_opengles_render_draw();
+
+        /* Swap front and back buffers */
+        glfwSwapBuffers(dsc->window);
+
+        glfwPollEvents();
+    }
+}
+
+static void glfw_error_cb(int error, const char * description)
+{
+    fprintf(stderr, "GLFW Error %d: %s\n", error, description);
+}
+
+static void window_close_callback(GLFWwindow * window)
+{
+    deiniting = true;
+}
+
+static void key_callback(GLFWwindow * window, int key, int scancode, int action, int mods)
+{
+    if(key == GLFW_KEY_ESCAPE && action == GLFW_PRESS) {
+        deiniting = true;
+    }
+}
+
+void _lv_opengles_mouse_btn_handler(lv_display_t * disp, int btn_down);
+
+static void mouse_button_callback(GLFWwindow * window, int button, int action, int mods)
+{
+    if(button == GLFW_MOUSE_BUTTON_LEFT) {
+        double xpos, ypos;
+        glfwGetCursorPos(window, &xpos, &ypos);
+        lv_display_t * disp = lv_display_get_default();
+        if(action == GLFW_PRESS) {
+            // LV_LOG_INFO("Mouse position: (%f, %f)\n", xpos, ypos);
+            _lv_opengles_mouse_btn_handler(disp, 1);
+        }
+        else if(action == GLFW_RELEASE) {
+            _lv_opengles_mouse_btn_handler(disp, 0);
+        }
+    }
+}
+
+void _lv_opengles_mouse_move_handler(lv_display_t * disp, int x, int y);
+
+static void mouse_move_cllback(GLFWwindow * window, double xpos, double ypos)
+{
+    // LV_LOG_INFO("Mouse position: (%f, %f)\n", xpos, ypos);
+    lv_display_t * disp = lv_display_get_default();
+    _lv_opengles_mouse_move_handler(disp, (int)xpos, (int)ypos);
+}
+
+static void framebuffer_size_callback(GLFWwindow * window, int width, int height)
+{
+    glViewport(0, 0, width, height);
+    lv_display_t * disp = lv_display_get_default();
+    if(disp == NULL) {
+        return;
+    }
+    lv_opengles_window_t * dsc = lv_display_get_driver_data(disp);
+    dsc->ignore_size_chg = 1;
+    lv_display_set_resolution(disp, width / dsc->zoom, height / dsc->zoom);
+    dsc->ignore_size_chg = 0;
+    lv_refr_now(disp);
+}
+
+static void texture_resize(lv_display_t * disp)
```

Comment:
It will help a lot if you can provide more info about this bug, since I cannot reproduce it on my machine. I did find another issue of memory, but maybe it has no relationship with this bug.

Reply:
I use an ubuntu 22 in virtualbox, and I can change the size of window with mouse:
![image](https://github.com/lvgl/lvgl/assets/24490390/1c18b2b6-15c5-449c-8094-faac78c758a8)
![image](https://github.com/lvgl/lvgl/assets/24490390/358b13d2-5efd-41e7-a4d1-d2f057950ce3)

I will check the resize function, to see what is not done correctly. Maybe you can let me know which system you are using, so that I can check if it has any relationship with different system.

Reply:
I'm on Linux Mint 21.3

Reply:
Please check if this issue is fixed with the latest code. I can produce it in a real windows system, it doesn't crash, while it does show same picture which becseya shown above. It seems system in virtualbox has a different implementation of opengl, or a fake gpu, so the issue can only be reproduced in real system with a real gpu. The reason I see is the data pass to gpu is not aligned correctly, so the picture will not shown correctly, and function glTexImage2D will crash. @becseya @kisvegabor